src/transaction/log_page_buffer.c

```diff
@@ -1269,15 +863,7 @@ logpb_set_dirty (THREAD_ENTRY * thread_p, LOG_PAGE * log_pgptr, int free_page)
     }
 #endif /* CUBRID_DEBUG */
 
-  START_EXCLUSIVE_ACCESS_LOG_PB (rv, thread_p);
-
   bufptr->dirty = true;
-  if (free_page == FREE)
```

Comment:
@funkydvd 

Please notice that we have a case of `FREE` for `free_page`. 
Please find `logpb_end_append`.